app/uk/gov/hmrc/selfassessmentrefundfrontend/controllers/refundRequestJourney/SelectRepaymentAmountController.scala

```diff
@@ -203,11 +203,6 @@ class SelectRepaymentAmountController @Inject() (
 
   private def logAndReturnErrorPage(method: String, message: String = "Not enough data in mongo to proceed.")(implicit request: BarsVerifiedRequest[_]): Future[Result] = {
     logger.error(s"[SelectRepaymentAmountController][$method] $message ${request.journey.toLogMessage}")
-    errorHandler.standardErrorTemplate(
-      Messages("global.error.InternalServerError500.title"),
-      Messages("global.error.InternalServerError500.heading"),
-      Messages("global.error.InternalServerError500.message")
-    )
-      .map(InternalServerError(_))
+    errorHandler.standardErrorTemplate("", "", "").map(InternalServerError(_))
```

Comment:
Hmm, just to clarify, by moving away from relying on the messages as parameters, unhandled errors **will** lose clarity in their message.

e.g. Unhandled 404 scenario would have used the following message keys:
global.error.pageNotFound404.title=Page not found - 404
global.error.pageNotFound404.heading=This page can’t be found
global.error.pageNotFound404.message=Please check that you have entered the correct web address.

It will now just give the try again later spiel. Ideally we would want to limit the scope of this change to just the fallbackClientErrorTemplate and the internalServerErrorTemplate. Whether that's by overriding their templates specifically or adding logic into the base standardErrorTemplate override.

Reply:
Either that or just confirm that Design is happy with that change in behaviour/or confirm I'm not misunderstanding how the error template works.